modules/aws-shield/remote-state.tf

```diff
@@ -0,0 +1,9 @@
+module "alb" {
+  count   = length(var.alb_names) > 0 ? 0 : 1
+  source  = "cloudposse/stack-config/yaml//modules/remote-state"
+  version = "1.5.0"
+
+  component = "eks/alb-controller-ingress-group"
```

Comment:
nitpick: `eks` component name changes sometimes. this should be a variable but probably isnt required right now